.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -30,6 +30,10 @@ about: Report broken or incorrect behaviour
 
 ### System Information
 
-- discord.py version:
-- Python version:
-- Operating system:
+<!-- Do `python -m discord -v` and paste this information.-->
```

Comment:
```suggestion
<!-- Run `python -m discord -v` and paste this information.-->
```

Reply:
Sounds good to me. Might need `at the command line` after the command though.